cumulusci/utils/salesforce/count_sobjects.py

```diff
@@ -0,0 +1,46 @@
+import typing as T
+
+from simple_salesforce import Salesforce
+
+from cumulusci.utils.http.multi_request import CompositeParallelSalesforce
+from cumulusci.utils.iterators import partition
+
+
+class ObjectCount(T.NamedTuple):
```

Comment:
What's the intent behind subclassing `T.NamedTuple`? Preferable to a plain old object? (Someday we'll be able to use dataclasses).

Reply:
It really is a named tuple. A tuple of return values that I gave names to for readability purposes. Dataclassses would be way overkill, performance-wise.